glambie/const/density_uncertainty.py

```diff
@@ -0,0 +1,28 @@
+"""
+Density uncertainty in relation to the length of the survey period.
+"""
+
+
+def get_density_uncertainty_over_survey_period(time_period_in_fractional_years) -> int:
+    """
+    Lazy implementation of a density uncertainty catalogue at different time resolutions,
+    approximately following Huss (2013, Fig. 4d)
+
+    Parameters
+    ----------
+    time_period_in_fractional_years : float
+        time period of desired density uncertainty
+
+    Returns
+    -------
+    int
+        Density uncertainty in kg/m^3
```

Comment:
can we be specific about what we mean by "uncertainty"? There are quite a few different ways of characterising uncertainty. Do we actually mean [Standard Error](https://en.wikipedia.org/wiki/Standard_error) here?

Reply:
as discussed in g-chat, will rename to uncertainties. ALso now refering to ATBD in the docstrings